README.md

```diff
@@ -4,38 +4,36 @@
+A command line tool that assesses the quality of a set of data sources (e.g.: CSV files of open data published by a government).
 
 ## What's it about?
 
-The `dq` (alias: `dataquality`) CLI is a batch processor
-
-is for administering data that belongs to a Spend Publishing
-
-Dashboard (dashboard data is in itself a github repository:
-<a href="https://github.com/okfn/spd-data-example">see the example repository here</a>).
+The `dq` (alias: `dataquality`) CLI is a tool to create and manage a [Data Package](http://specs.frictionlessdata.io/data-packages/) 
+from a given source of data that can be used by [Data Quality Dashboard](https://github.com/frictionlessdata/data-quality-dashboard).
+The quality assessment is done using [GoodTables](http://goodtables.readthedocs.io/en/latest/index.html) and [can be configured](#quality-config).
 
 The proposed workflow is this:
 
-* A deployer/administrator runs the validation over a set of data sources at regular intervals
-* The data is managed in a git repository (eg), which the developer has locally
-* The deployer/administrator should have a config file for each Spend Publishing
-Dashboard she is administering
-* The deployer/administrator, or possibly content editor, occasionally updates
-the `sources.csv` file in the data directory with new data sources
-* Periodically (once a month, once a quarter), the deployer/administrator does
+* An administrator creates a folder for a given project which will be equivalent to a data package.
+* The administrator creates a `source_file` and `publisher_file`:
+    * By using the [generate command](#generate).
+    * By using custom scripts ([see this example](https://github.com/okfn/data-quality-uk-25k-spend)).
+    * In any other way that is in sync with the [schema](#schema).
+* The administrator updates the [configuration file](#config) to reflect the structure of the data package 
+and optionally to configure the quality assessment.
+* The administrator [runs the validation](#run) over the set of sources.
+* The data is managed in a git repository (or other version control system), which the administrator has locally
+* The administrator [deploys](#deploy) the data package to a central data repository (ex: GitHub)
+* The administrator [updates the configuration](https://github.com/frictionlessdata/data-quality-dashboard#configure-database)
+of the corresponding Data Quality Dashboard instance
+* The administrator, or possibly content editor, occasionally updates
+the `source_file` file in the data directory with new data
+* Periodically (once a month, once a quarter), the administrator runs
 `dq run /path/to-config.json --deploy`. This builds a new set of results for the data,
-and deploys the updated data back to the central data repository (i.e: GitHub)
-* As the Spend Publishing Dashboard is a pure client-side application, as soon as updated
```

Comment:
Not sure if we should call it `Data Quality Dashboard`(as in the GH repo) or `Spend Publishing Dashboard`. I assumed the scope exceeds "spend publishing" now, but I may be wrong.